packages/plugins/actions/actions-core/src/actions/DisplayMessage.js

```diff
@@ -0,0 +1,27 @@
+/*
+  Copyright 2020-2021 Lowdefy, Inc
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+import { type } from '@lowdefy/helpers';
+
+function DisplayMessage({ methods: { displayMessage }, params }) {
+  let defaultParams = params;
```

Comment:
Looking at this, lets please implement the following:
- If params are `null` or `undefined` => display message with `Sucess` as content.
- If params is an object, display message with `Sucess` as content if `params.content` is of none type.
- If params is not and object or none type, throw a useful error message.

Please also add tests for these cases, especially a test for a falsly content value like `""`, which should not show the default.